src/jobs/ega/services/permissions.ts

```diff
@@ -31,6 +41,34 @@ import {
   createRevokePermissionRequest,
 } from '../utils';
 
+/**
+ * Parse completionStatus for a reconciliation step, based on the number of successfully processed items vs total expected
```

Comment:
added an `INCOMPLETE` status to reflect a scenario where no errors are recorded, but the expected totals do not line up, i.e. the number of permissions for a dataset does not match the approved users count. This could mean a request was dropped somewhere in the process, but did not register as an error, like a failed retry.